environment.yml

```diff
@@ -28,6 +28,7 @@ dependencies:
   - flask-cors==4.0.0
   - redis-py==5.0.1
   - botocore>=1.33.10 # Minimum version that is compatible with python >= 3.10 is botocore>=1.13.0
+  - scrapy==2.11.1
```

Comment:
I think we should lock down the Scrapy version in `newzealidar` instead of here, as the digital twin doesn't use Scrapy. Although Sam has already merged this Scrapy version change into the Otakaro master branch, I plan to revert it once the version is locked down in `newzealidar`.

Reply:
I agree with this. Leaving this PR up temporarily until we sort it out in `newzealidar`

Reply:
PR included in #166